webpackConfigLoader.js

```diff
@@ -51,7 +51,7 @@ const configLoader = (configPath) => {
   // Some test environments might not have the NODE_ENV set, so we'll have fallbacks.
   const configEnv = env.NODE_ENV || env.RAILS_ENV || 'development';
   const ymlConfigPath = join(configPath, 'webpacker.yml');
-  const settings = safeLoad(readFileSync(ymlConfigPath, 'utf8'))[configEnv];
+  const settings = load(readFileSync(ymlConfigPath, 'utf8'))[configEnv];
```

Comment:
@ahangarha can you explain why we had to stop using safeLoad?

Reply:
Sure. That is because of the [new changes in `js-yaml` v4.0.0](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md#400---2021-01-03).

> Breaking: removed `safe*` functions. Use `load`, `loadAll`, `dump` instead which are all now safe by default.

They are all safe by this version.

Reply:
Figure out why we still have this file and if we can delete this.

Reply:
@justin808 @Judahmeek 

We don't use this file anymore, but I am not sure if no other project uses this as config loader. It is very unlikely but still...

It is included in out npm package and removing it is a breaking change.

What about putting this file in the queue for removal in the next major release and for now, we show a deprecation message for using it?